tests/codegen/ccode/scripts/ifs.py

```diff
@@ -0,0 +1,99 @@
+from numpy import zeros
+
+x = 0
+y = 0
+z = 3
+b = zeros(64,'int')
```

Comment:
See above. This line and all lines using it must be removed as arrays are not yet handled in C